app/src/main/java/com/iic/shopingo/ui/trip_flow/data/Request.java

```diff
@@ -9,16 +9,24 @@
  * Created by asafg on 03/03/15.
  */
 public class Request implements Parcelable {
+  public final static int STATUS_PENDING = 0;
+  public final static int STATUS_ACCEPTED = 1;
+  public final static int STATUS_DECLINED = 2;
+  public final static int STATUS_PAID = 3;
```

Comment:
All the model-related stuff is going to change anyway once we have unified models.